examples/rust/get_started/examples/04-unix-domain-socket-transport-responder.rs

```diff
@@ -0,0 +1,21 @@
+// This node starts a uds listener and an echoer worker.
+// It then runs forever waiting for messages.
+
+use hello_ockam::Echoer;
+use ockam::{Context, Result};
+use ockam_transport_uds::UdsTransport;
+
+#[ockam::node]
+async fn main(ctx: Context) -> Result<()> {
+    // Initialize the Uds Transport.
+    let uds = UdsTransport::create(&ctx).await?;
+
+    // Create a Uds listener and wait for incoming connections.
+    uds.listen("/tmp/ockam-example-echoer").await?;
```

Comment:
FYI: we should prefer first setting up everything on our node (in this case - starting an `Echoer`) and only then start listening for incoming connections. But since other examples have it wrong, this one should stay wrong as well :D

Reply:
Noted! 
Is there a specific reason behind this? (Outside of just a more logical sequence and potential race conditions.)

Reply:
I think race conditions is the main reason